packages/wdio-browserstack-service/README.md

```diff
@@ -179,6 +179,22 @@ Automatically set the BrowserStack Automate session status (passed/failed).
 Type: `Boolean`<br />
 Default: `true`
 
+### buildIdentifier
+
+**buildIdentifier** is a unique id to differentiate every execution that gets appended to
+buildName. Choose your buildIdentifier format from the available expressions:
+
+* ${BUILD_NUMBER}: Generates an incremental counter with every execution
+* ${DATE_TIME}: Generates a Timestamp with every execution. Eg. 05-Nov-19:30
+
```

Comment:
* Build Identifier supports usage of either or both expressions along with any other characters enabling custom formatting options.